ledger-core/testutils/pulsewatcher/configuration/configuration.go.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
Filename is `ledger-core/testutils/pulsewatcher/configuration/configuration.go.go`
Extra `.go` suffix?